### PR TITLE
add tiup ssh handshake failed faq

### PR DESCRIPTION
**Pull request by WentaoJin**
add tiup ssh handshake failed faq

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/3741

### CODE REVIEW

File: tiup/tiup-faq.md [hunk 1]
```diff
@@ -38,3 +38,11 @@ TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境
 ## 是否可以检测不同集群的端口和目录冲突？
 
 如果不同集群是由同一个 TiUP 中控机部署的，会在部署和扩容时进行检测，如果属于不同的 TiUP 中控机，目前不支持检测。
+
+## 集群部署期间，TiUP 收到报错 ssh: handshake failed: read tcp 10.10.10.34:38980 -> 10.10.10.34:3600: read: connection reset by peer
```
Comment (TomShawn) [inline tiup/tiup-faq.md:42]:
```suggestion
## 集群部署期间，TiUP 收到报错 `ssh: handshake failed: read tcp 10.10.10.34:38980 -> 10.10.10.34:3600: read: connection reset by peer`
```

File: tiup/tiup-faq.md [hunk 2]
```diff
@@ -38,3 +38,11 @@ TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境
...
+该报错可能是因为 TiUP 默认并发超过 ssh 默认最大连接数导致，可尝试编辑加大默认连接数，然后重启 sshd 服务解决：
```
Comment (TomShawn) [inline tiup/tiup-faq.md:43]:
```suggestion
该报错可能是因为 TiUP 默认并发超过 ssh 默认最大连接数导致的，可尝试加大默认连接数，然后重启 sshd 服务解决：

```

**Comment by TomShawn** [comment]
@WentaoJin Does this change apply to `master`?

**Comment by WentaoJin** [comment]
> @WentaoJin Does this change apply to `master`?

sure，it only can meet ssh error when tiup is using

**Comment by WentaoJin** [comment]
> @WentaoJin Does this change apply to `master`?

sure，it only can meet ssh error when tiup is using @TomShawn

File: tiup/tiup-faq.md [hunk 3]
```diff
@@ -38,3 +38,12 @@ TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境
 ## 是否可以检测不同集群的端口和目录冲突？
 
 如果不同集群是由同一个 TiUP 中控机部署的，会在部署和扩容时进行检测，如果属于不同的 TiUP 中控机，目前不支持检测。
+
+## 集群部署期间，TiUP 收到报错 `ssh: handshake failed: read tcp 10.10.10.34:38980 -> 10.10.10.34:3600: read: connection reset by peer`
+该报错可能是因为 TiUP 默认并发超过 ssh 默认最大连接数导致的，可尝试加大默认连接数，然后重启 sshd 服务解决：
```
Comment (TomShawn) [inline tiup/tiup-faq.md:43]:
```suggestion

该报错可能是因为 TiUP 默认并发超过 ssh 默认最大连接数导致的，可尝试加大默认连接数，然后重启 sshd 服务解决：
```

**Comment by TomShawn** [comment]
> > @WentaoJin Does this change apply to `master`?
> 
> sure，it only can meet ssh error when tiup is using @TomShawn

Could you create another PR to make the change in `master`? This PR can only be manually cherry-picked to `master`.

**Comment by WentaoJin** [comment]
> > > @WentaoJin Does this change apply to `master`?
> > 
> > 
> > sure，it only can meet ssh error when tiup is using @TomShawn
> 
> Could you create another PR to make the change in `master`? It is recommended to create PRs based on `master` and use the cherry-pick labels to easily cherry-pick changes to other release branches. The other way around currently does not work out.

ok，has created PR to master @TomShawn

**Pull request merged**